### DISCUSSION TITLE
Prepend chip to q-input

### DISCUSSION THREAD

**Discussion by BalazsMakrai** [quasarframework/quasar#15078] [General - Components / Directives / etc]
I've got this code:
```
 <q-input v-model="toText" dense color="grey" :label="t('mailing.to')" class="text-caption row" stack-label
      @keyup.enter="addNewChip">
      <template v-slot:prepend>
        <q-chip size="sm" outline color="primary" removable v-for="chip in chips" :key="chip">{{ chip
        }}</q-chip>
      </template>
    </q-input>
```
and the function is:
```
  const addNewChip = () => {
      chips.value.push(toText.value);
      toText.value = '';
    };
```

What I want to achive is if the enter is pressed a new chip will be added to the input field. The logic works fine, but the layout is very strange..

![1](https://user-images.githubusercontent.com/64318333/206503928-1310493a-f62b-42f4-a731-3ae7a5698a9b.png)

Is there any way to show the "To" text in the top left corner and show the chips below it?

**Comment by pdanpdan** [comment]
With QInput I'm not sure (make a codepen so we can play with css for label)
If it's not working then you should try to put the chips and input in a QField
